.github/workflows/dart.yml

```diff
@@ -46,11 +46,19 @@ jobs:
             dart test -p vm --coverage=coverage
             dart pub run coverage:format_coverage --lcov --in=coverage --out=coverage/lcov.info --packages=.packages --report-on=lib
 
-        - name: Build example
+        - name: Install webdev
+          run: dart pub global activate webdev
+
+        - name: Build examples
           run: |
             cd example
             dart pub get
             dart compile exe bin/example.dart
+            cd ..
+            cd example_web
+            dart pub get
+            dart pub get
+            dart run webdev build
```

Comment:
getting error: The pubspec.lock file has changed since the .dart_tool/package_config.json file was generated, please run "pub get" again. - only on Windows
@bruno-garcia do you have a windows machine? would you like to help me out?